src/aind_hcr_data_transformation/zeiss_job.py

```diff
@@ -129,61 +128,46 @@ def _write_stacks(self, stacks_to_process: List) -> None:
             acquisition_path=acquisition_path
         )
 
+        # Output path for VAST
+        output_path = Path(self.job_settings.output_directory)
+        bucket_name = None
+
+        # If s3_location is provided, we define the bucket name
+        # and parse the output path to the prefix
+        if self.job_settings.s3_location:
+            parsed = urlparse(self.job_settings.s3_location)
+            bucket_name = parsed.netloc
+            output_path = Path(parsed.path.lstrip("/"))
+
         # Converting CZI tiles to Multiscale OMEZarr
         for stack in stacks_to_process:
             logging.info(f"Converting {stack}")
             stack_name = stack.stem
 
-            output_path = Path(self.job_settings.output_directory)
-
             msg = (
                 f"Voxel resolution ZYX {voxel_size_zyx} for {stack} "
-                f"with name {stack_name} - output: {output_path}"
+                f"with name {stack_name} - output: {output_path} "
+                f"with bucket name: {bucket_name}"
             )
             logging.info(msg)
 
             czi_stack_zarr_writer(
                 czi_path=str(stack),
                 output_path=output_path,
                 voxel_size=voxel_size_zyx,
-                final_chunksize=self.job_settings.chunk_size,
+                shard_size=self.job_settings.shard_size,
```

Comment:
Let's definitely have some docs which help people define reasonable shard and chunk sizes somewhere.